### DISCUSSION TITLE
Multi Tenancy team_id and team relationship required in every resources.

### DISCUSSION THREAD

**Discussion by sc-hero** [filamentphp/filament#8100] [General]
For Manage Team, I created team-resource but show the not have team relationship, So, I add team modal,

 public function team(): BelongsTo
{
    return $this->belongsTo(Team::class);
}

And next the error ,Undefined column: 7 ERROR: column teams.team_id does not exist LINE 1: select count(*) as aggregate from "teams" where "teams"."tea... ^ HINT: Perhaps you meant to reference the column "teams.team_code".

So, Above code updated,

 public function team(): BelongsTo
{
    return $this->belongsTo(Team::class, 'id', 'team_id');
}

All things is fine!

but, In the team resource, not show the any data related team data.Actually, It have 2 teams data dd(Team::all());

How to make ignoring the team_id in team model or how to fix?

**Comment by danharrin** [comment]
The team resource only ever shows 1 team, right? So maybe use a custom page with a form on it

**Reply by sc-hero** [reply]
Yes Right, Can I ignore team_id in a resource?
